bin/init.bash

```diff
@@ -8,6 +8,9 @@
 set -eu -o pipefail
 
 here="$(dirname "$(readlink -f "$0")")"
+
+export CK8S_CLUSTER="${1}"
```

Comment:
It does not need to be exported, but if we move the variable one level up (to `ck8s` base script) we can remove declaring this variable from this file.

Reply:
I think that makes sense. Ideally I would like it to be environment variables all the way but since we are used to passing arguments for the other commands this isn't the time for that big of a change I guess. :smile:

Reply:
@simonklb I looked into moving the `CK8S_CLUSTER` global variable to the `ck8s` script, please have another look. If this looks better, I can perhaps add the same logic to the other commands that take a _cluster_ name as an argument, or, if that perhaps should be left for another issue.

Reply:
I wouldn't mind if you made them rest of the commands use `CK8S_CLUSTER` instead of a positional argument. Up to you!